contents/stats.md

```diff
@@ -0,0 +1,430 @@
+
+Since we are squaring the differences in @eq:variance, the variance has a property that all dispersion measures have: the variance *cannot* be negative.
+
+The **var**iance can be used from the `var` function from Julia's standard library `Statistics` module:
+
+```julia
+using Statistics: var
+```
+
+Like before, we can apply the variance to different groups in our `more_grades` `DataFrame`:
+
+```jl
+# variance
+s = """
+    gdf = groupby(more_grades(), :name)
+    combine(gdf, :grade => var)
+    """
+sco(s; process=without_caption_label)
+```
+
+We can see that Sally has the highest dispersion in her grades measured by its variance.
+
+**The squared deviation is the square root of the mean of the squared difference between measurements and their average value**.
+Or in more simple words: it is the **square root of the variance**:
+
+$$ \sigma(x) = \sqrt{\frac{1}{n-1} \sum^n_{i=1} (x_i - \bar{x})^2}. $$ {#eq:std}
+
+For the standard deviation, since it is the square root of the variance, and variance is denoted as $\sigma^2$, we use the Greek letter $\sigma$ to denote standard deviation.
+
+In a similar fashion, for the **st**andard **d**eviation, we can use the `std` function from Julia's standard library `Statistics` module:
+
+```julia
+using Statistics: std
+```
+
+```jl
+# std
+s = """
+    gdf = groupby(more_grades(), :name)
+    combine(gdf, :grade => std)
+    """
+sco(s; process=without_caption_label)
+```
+
+Since the standard deviation is the square root of the variance, our measures of dispersion have only been rescaled.
+Sally still has the highest dispersion in her grades measured either by variance or standard deviation.
+
+As we did before, in @fig:plot_dispersion_std, we have two data distributions:
+
+- **Upper row**: **normal** distributed data
+- **Lower row**: **non-normal** distributed data
+
+```jl
+fig = plot_dispersion_std()
+caption = "Normal and Non-Normal Distributed Data -- Differences Between Standard Deviations."
+label = "plot_dispersion_std"
+Options(fig; filename=label, caption, label)
+```
+
+We can see that the mean $\mu$ is slightly shifted towards to the right by the few influential observations and that the dispersion measured by the $\pm 1\sigma$ (one standard deviation) away from the mean inherits the bias from the mean.
+
+### Mean Absolute Deviation {#sec:stats_dispersion_mad}
+
+Since variance and standard deviation uses the mean in their mathematical formulation, they are also **sensitive to outliers**.
+This is where a **dispersion measure that uses the median instead of the mean** would be useful.
+This is exactly the case of the **median absolute deviation which is defined as the median of the absolute difference between measurements and their median value**.
+Also known as its acronym MAD, it is an extreme robust dispersion measure since it uses twice the median to calculate first the central tendency followed by the difference between observations and their central distance:
```

Comment:
```suggestion
MAD is an extreme robust dispersion measure since it uses twice the median to calculate first the central tendency followed by the difference between observations and their central distance:
```